File: Linear Regression/octave/featureNormalize.m
function [X_norm, mu, sigma] = featureNormalize(X)
  % Returns a normalized version where the mean
  % value of each feature is 0 and the standard deviation is 1.
mu = mean(X);
sigma = std(X);
a = X - mu;
X_norm = a./ sigma;

end

File: Linear Regression/2.Multivariate/python/featureNormalize.py
import numpy as np
def featureNormalize(X):
#   Returns a normalized version where
#   mean value of each feature is 0 and the standard deviation is 1.
    mu = np.mean(X)
    sigma = np.std(X)
    a = X - mu
    X_norm = a / sigma
    return X_norm, mu, sigma